apisix/init.lua

```diff
@@ -372,7 +372,7 @@ function _M.handle_upstream(api_ctx, route, enable_websocket)
         local upstream = apisix_upstream.get_by_id(up_id)
         if not upstream then
             if is_http then
-                return core.response.exit(502)
+                return core.response.exit(core.ctx, 502)
```

Comment:
Sorry, what does the core.ctx here mean? It should be `ctx` (or sometimes called `api_ctx`).